README.md

```diff
@@ -43,6 +43,18 @@ To upgrade to the latest (unstable) version, run
 pip install --upgrade git+https://github.com/cornellius-gp/gpytorch.git
 ```
 
+### ArchLinux Package
+Note: The ArchLinux package is not tested by the GPyTorch developers.
+
+GPyTorch is also available on the [ArchLinux User Repository](https://wiki.archlinux.org/index.php/Arch_User_Repository) (AUR).
+You can install it with an [AUR helper](https://wiki.archlinux.org/index.php/AUR_helpers), like [`yay`](https://aur.archlinux.org/packages/yay/), as follows:
+
+```bash
+yay -S python-gpytorch
+```
+To discuss any issues related to this package refer to the comments section of
```

Comment:
We should clarify that this only pertains to issues with the AUR package/installation.

Let's say `To discuss any issues specific to this AUR package refer to the comments section of...`

Reply:
Yeah, its good to be explicit.